includes/class-wc-payments.php

```diff
@@ -1378,7 +1386,9 @@ function wcpay_show_old_woocommerce_for_norway_notice() {
 	 */
 	public static function remove_woo_admin_notes() {
 		if ( defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '4.4.0', '>=' ) ) {
-			self::$remote_note_service->delete_notes();
+			if ( class_exists( 'WC_Payments_Remote_Note_Service' ) && self::$remote_note_service instanceof WC_Payments_Remote_Note_Service ) {
```

Comment:
We don't want developers thinking they need to do this everywhere they call a method. A comment here explaining why this is a special case would be useful.

I know it's small change, but it's technically unrelated to the other change as well right? Would this be better in its own PR?